libdnf/dnf-context.h

```diff
@@ -293,15 +293,32 @@ gboolean         dnf_context_reset_all_modules          (DnfContext * context,
  * @module_specs: Module specs that should be enabled
  * @error: Error
  *
- * Enable mudules, recalculate module filtration, but do not commit modular changes.
- * To commit modular changes it requires to call dnf_context_run()
+ * Enable modules, recalculate module filtration, but do not commit modular changes.
+ * To commit modular changes, call dnf_context_run().
  * Returns FALSE when an error is set.
  *
  * Since: 0.55.0
  **/
 gboolean         dnf_context_module_enable              (DnfContext * context,
                                                          const char ** module_specs,
                                                          GError ** error);
+
+/**
+ * dnf_context_module_install:
+ * @context: DnfContext
+ * @module_specs: Module specs that should be installed
+ * @error: Error
+ *
+ * Enable modules and mark for installation but do not commit modular changes.
+ * To commit modular changes, call dnf_context_run().
+ * Returns FALSE when an error is set.
+ *
+ * Since: 0.63.0
```

Comment:
You'll need to bump the version in `CMakeLists.txt` and the spec file accordingly.